images/Dockerfile.broker

```diff
@@ -1,10 +1,17 @@
-from golang:1.10-alpine3.8 as builder
+from golang:1.11-alpine3.8 as builder
 workdir /go/src/github.com/cloudfoundry-incubator/blockhead
 copy . .
 run go build -v -o /broker ./cmd/broker
 
-from alpine:3.8
+
+from node
 expose 3333
+RUN npm install --production --quiet cli-flags
```

Comment:
LGTM. however testing `pusher.js` recently, it appears that cli-flags is not maintained anymore and may not work so we may have to switch to a different node flags library. 

If this is the case, we are going to have problem with the ruby version of the broker too.

Reply:
turns out my problem was with the version of node install on my machine. I was running the very old version `4.6.2`. upgrading to `0.10.10` fixed the initial error I was getting.

That said, can we lock down the version of our binaries.

node: `>0.9` ideally `0.10.10` or the latest that works
geth: `1.8.2` worked for me in the past but newer versions could be available.

what other binaries do we have to lock down?

Reply:
The docker image here is node 10.
I did not get a response earlier when I asked if we needed a specific version of node.
I am not familiar with how node does versions to know what to do.